builder/bundles.go

```diff
@@ -493,15 +547,79 @@ func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *b
 			rpmList = append(rpmList, p)
 		}
 
+		startTime := time.Now()
 		// Retry RPM downloads to avoid timeout failures due to slow network
 		_, err = downloadRpms(packagerCmd, rpmList, baseDir, downloadRetries)
 		if err != nil {
 			return err
 		}
+		fmt.Println("download time : ", time.Since(startTime).String())
+		elapsedTime = elapsedTime + time.Since(startTime)
+
+		path := baseDir + "/var/cache/yum/clear/"
+		newPath, err := filepath.Glob(filepath.Join(path, "clear-*"))
+		if err != nil {
+			fmt.Print("cant find path to rpm")
+			return err
+		}
+		if len(newPath) > 0 {
```

Comment:
if you swap `if` and `else` conditions here, you wont need an `else` block anymore. It will make the code look cleaner.